benchmarks/snakeCase.bench.ts

```diff
@@ -0,0 +1,15 @@
+import { bench, describe } from 'vitest';
+import { snakeCase as snakeCaseToolkit } from 'es-toolkit';
+import { snakeCase as snakeCaseLodash } from 'lodash';
+
+describe('snakeCase', () => {
+  bench('es-toolkit/snakeCase', () => {
+    const str = 'camleCase';
```

Comment:
```suggestion
    const str = 'camelCase';
```
Typo here :)

Reply:
Oh, I will fix the typo and open a pr. Thanks for the thorough code review!

Reply:
resolved in pr #157 :)